client/pages/item/_itemId/index.vue

```diff
@@ -392,3 +392,9 @@ export default Vue.extend({
   }
 });
 </script>
+
+<style>
```

Comment:
I think we want this scoped to this component only?

Reply:
yep, but is there a way to inject custom style to Vuetify component?

Reply:
You don't have to, just use `<style scoped>` :)